spec/checkout_spec.rb

```diff
@@ -632,6 +632,23 @@
     ).to have_been_made.once
   end
 
+  # must be created manually because every field in the response is an array
+  it "makes a LIVE payment_methods call" do
```

Comment:
```suggestion
  it "makes a LIVE paymentMethods call" do
```
Should be like this similar to [`storedPaymentMethods`](https://github.com/Adyen/adyen-ruby-api-library/blob/af6fba95f2a4d889c062600a5572b9abec11d482/spec/checkout_spec.rb#L565)?